events/core/types.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"time"
+
+	kube_api "k8s.io/kubernetes/pkg/api"
+)
+
+type EventBatch struct {
+	// When this batch was created.
+	Timestamp time.Time
```

Comment:
We have timestamp here:
https://github.com/kubernetes/heapster/blob/heapster-scalability/metrics/core/types.go#L76